CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# Contributing
+
+If you discover issues, have ideas for improvements or new features,
```

Comment:
Can you add

`emacs-lsp` is a very active organization so before you start working on an issue make sure that there is corresponding issue and drop a note that you start working on that in order to avoid collisions. Feel free to ping the mantainers in `gitter` chat or in the issue report if you need help/clarification on how the things work.

Reply:
Sure, done!